__tests__/app.test.js

```diff
@@ -192,4 +192,86 @@ describe("/api/articles/:article_id", () => {
             });
         });
     });
-});
\ No newline at end of file
+});
+
+describe("/api/articles/:article_id/comments", () => {
+    describe("GET", () => {
+        test("Returns 'Status: 200' if no error in path", () => {
+            return request(app).get('/api/articles/1/comments').expect(200)
+        });
```

Comment:
this test is covered in the next test, so there is no need for it.

Reply:
Thanks Niamh, will refactor! :)